games-action/btanks/btanks-0.9.8083.recipe

```diff
@@ -32,7 +32,7 @@ REQUIRES="
 	haiku$secondaryArchSuffix
 	lib:libexpat$secondaryArchSuffix
 	lib:libGLU$secondaryArchSuffix
-	lib:liblua$secondaryArchSuffix
+	lib:liblua$secondaryArchSuffix >= 5.3
```

Comment:
Doesn't hurt, but version string should only be needed for the devel requirement.